README.md

```diff
@@ -289,135 +299,242 @@ func main() {
    ]
 }`
 
-    definition := make(map[string]interface{})
-    _ = json.Unmarshal([]byte(jsonStr), &definition)
-
-    client := twilio.NewRestClient(accountSid, authToken)
-    params := &openapi.CreateFlowParams{
-        Definition: &definition,
-    }
-    params.SetCommitMessage("commit")
-    params.SetFriendlyName("Studio flow from Go")
-    params.SetStatus("draft")
-
-    resp, err := client.StudioV2.CreateFlow(params)
-    if err != nil {
-        fmt.Println(err.Error())
-    } else {
-        fmt.Println(*resp.Sid)
-    }
+	definition := make(map[string]interface{})
+	_ = json.Unmarshal([]byte(jsonStr), &definition)
+
+	client := twilio.NewRestClient(accountSid, authToken)
+	params := &openapi.CreateFlowParams{
+		Definition: &definition,
+	}
+	params.SetCommitMessage("commit")
+	params.SetFriendlyName("Studio flow from Go")
+	params.SetStatus("draft")
+
+	resp, err := client.StudioV2.CreateFlow(params)
+	if err != nil {
+		fmt.Println(err.Error())
+	} else {
+		fmt.Println(*resp.Sid)
+	}
+}
+```
+
+### Using Paging
+
+This library also offers paging functionality. Collections such as calls and messages have `ListXxx` and `StreamXxx`
+functions that page under the hood. With both list and stream, you can specify the number of records you want to
+receive (limit) and the maximum size you want each page fetch to be (pageSize). The library will then handle the task
+for you.
+
+`List` eagerly fetches all records and returns them as a list, whereas `Stream` streams the records and lazily retrieves
+the pages as you iterate over the collection. You can also page manually using the `PageXxx` function in each of the
+apis.
+
+```go
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/twilio/twilio-go"
+	openapi "github.com/twilio/twilio-go/rest/api/v2010"
+	"os"
+)
+
+func main() {
+	accountSid := os.Getenv("TWILIO_ACCOUNT_SID")
+	authToken := os.Getenv("TWILIO_AUTH_TOKEN")
+	from := os.Getenv("TWILIO_FROM_PHONE_NUMBER")
+
+	client := twilio.NewRestClient(accountSid, authToken)
+
+	params := &openapi.ListMessageParams{}
+	params.SetFrom(from)
+	params.SetPageSize(20)
+	limit := 100
+
+	resp, err := client.ApiV2010.ListMessage(params, limit)
+	for record := range resp {
+		fmt.Println("Body: ", *resp[record].Body)
+	}
+
+	for {
```

Comment:
I don't think this needs to be wrapped in a for loop. `range` should iterate over the channel until it's closed.